java-frontend/src/main/java/org/sonar/plugins/java/api/IssuableSubscriptionVisitor.java

```diff
@@ -31,6 +31,11 @@
  */
 public abstract class IssuableSubscriptionVisitor extends SubscriptionVisitor {
 
+  @Override
+  protected void scanTree(Tree tree) {
+    throw new UnsupportedOperationException("IssuableSubscriptionVisitor should not drive visit of AST.");
```

Comment:
This need to be documented as public API change for next release (what about adding it to the ticket?)
Some (badly written) custom rules may start failing after the update because of it.